pkg/virt-operator/resource/apply/delete.go

```diff
@@ -498,6 +499,10 @@ func DeleteAll(kv *v1.KubeVirt,
 			log.Log.Errorf(castFailedFmt, obj)
 			return nil
 		}
+	}	
```

Comment:
Empty space detected:
```suggestion
	}
```

Reply:
@fossedihelm i didn't get it . Empty space ?

Reply:
There is an empty space/tab char after the parenthesis. That has to be eliminated, otherwise the CI will not pass.
The `make generate` should automatically drop it for you.

Reply:
@fossedihelm ok . Thank you

Reply:
You are adding empty space here and then removing it in the subsequent commit.
Please remove the space addition directly here.